pkg/apis/internal.acorn.io/v1/map.go

```diff
@@ -49,9 +47,8 @@ func (in *GenericMap) UnmarshalJSON(data []byte) error {
 
 func (in *GenericMap) DeepCopyInto(out *GenericMap) {
 	*out = map[string]interface{}{}
-	for k, v := range runtime.DeepCopyJSON(*in) {
-		(*out)[k] = v
-	}
+	data, _ := in.MarshalJSON()
+	_ = out.UnmarshalJSON(data)
```

Comment:
This is probably not the best way to go about this, but it does fix the problem.